src/childviewcontainer.js

```diff
@@ -124,6 +89,53 @@ Backbone.ChildViewContainer = (function (Backbone, _) {
     // Update the `.length` attribute on this container
     _updateLength: function(){
       this.length = _.size(this._views);
+
+      return this;
+    },
+    // To be used when avoiding call _updateLength
+    // When you are done adding all your new views
+    // call _updateLength
+    _add: function(view, customIndex){
+      var viewCid = view.cid;
+
+      // store the view
+      this._views[viewCid] = view;
+
+      // index it by model
+      if (view.model){
+        this._indexByModel[view.model.cid] = viewCid;
+      }
+
+      // index by custom
+      if (customIndex){
+        this._indexByCustom[customIndex] = viewCid;
+      }
+
+      return this;
+    },
+    // To be used when avoiding call _updateLength
+    // When you are done adding all your new views
+    // call _updateLength
+    _remove: function (view){
+      var viewCid = view.cid;
+
+      // delete model index
+      if (view.model){
+        delete this._indexByModel[view.model.cid];
+      }
+
+      // delete custom index
+      _.any(this._indexByCustom, function(cid, key) {
```

Comment:
I just realized that `_.any` isn't part of lodash.core.
Should I update it to `_.some`?